peercrawler_html_server_utils.py

```diff
@@ -0,0 +1,26 @@
+from json import loads
+from typing import Any
+
+
+def try_load_peers_details(path: str) -> list:
+    try:
+        with open(path) as file:
+            contents = file.read()
+
+        return loads(contents)
+
+    except FileNotFoundError:
+        print("Warning: file with peers details was not found.")
+        return []
+
+
+def get_node_details_from_id(node_id: str, node_details: list) -> dict | None:
+    for i in node_details:
+        try:
```

Comment:
This try except can be eliminated by using the getter function.
But it is a minor complaint. Happy to accept as it is too.